hbase-shell/src/main/ruby/shell/commands/list_regions.rb

```diff
@@ -147,30 +148,30 @@ def command(table_name, options = nil, cols = nil)
             end
 
             if size_hash.key?('SIZE')
-              if region_load.nil?
+              if region_metrics.nil?
                 region_store_file_size = ''
               else
-                region_store_file_size = region_load.getStorefileSizeMB.to_s.strip
+                region_store_file_size = region_metrics.getStoreFileSize.to_s.strip
```

Comment:
Shouldn't this be something like `region_metrics.getStoreFileSize.get(Size.Unit.MEGABYTE)`?

Reply:
It can be used but not needed since by default it returns size in MB. When I printed locally, it's coming as 0.0MB / 1.2MB etc. Should be fine to print?